src/styles/utils/_count.scss

```
@@ -0,0 +1,7 @@
+@keyframes count {
+  @for $i from 0 through 12 {
+    #{$i * 100% / 12} {
+      transform: rotate(calc((-90deg + 30 * #{$i}deg)));
+    }
+  }
```

it's a bit overwhelming
wouldn't be cleaner to define start and end %?

```suggestion
  0% {
    transform: rotate(0);
  }

  100% {
    transform: rotate(360deg);
  }
}
```